<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>148</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>301</width>
    <height>148</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Saving...</string>
  </property>
  <widget class="QLineEdit" name="Name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Select_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>PNG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MEL</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="OK_Button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
